### ISSUE TITLE
Determine how information will be collected, what will be needed on day one, and how it will evolve. 

### ISSUE THREAD

**Issue by OpenGlobe** [18F/nsf#166]
## User story
As a \<type of user\>, I want \<a goal\> so that \<benefit\>. 

## Acceptance criteria
- [ ] Select feedback mechanism that matches PIA, and what it can evolve into (just email, then form, smiley-face, etc. )
- [ ] Content release schedule?

**Comment by trinehar** [comment]
NSF will create a mailbox (beta-nsf-feedback@nsf.gov). Initially, a link to this email address is fine. More elaborate and structured feedback mechanisms will be created as the site evolves.

**Comment by OpenGlobe** [comment]
@trinehar is this still the email you'd prefer to use for feedback on the beta?

**Comment by trinehar** [comment]
Yes, beta-nsf-feedback@nsf.gov<mailto:beta-nsf-feedback@nsf.gov> should be used as the email link for requesting feedback.

idbcmsadmin@nsf.gov<mailto:idbcmsadmin@nsf.gov> is being set up for admin emails.
